apps/main-app/src/constants.js

```diff
@@ -93,6 +93,7 @@ export const NAV_MENU = [
       {
         label: 'JBrowse',
         route: '/jbrowse/?data=data%2FHomo%20sapiens',
+        external: true,
```

Comment:
Can this be added to both bluegenes and api links?

Reply:
Yeah, sure.

Reply:
I'm guessing that external here means external to react not necessarily external to the site.

Reply:
Yeah, specifically using ExternalLink component